scan/scan_agents.py

```diff
@@ -1,89 +1,154 @@
-# scan_agents.py
-
 from textwrap import dedent
 from crewai import Agent
 
 
 class PFCAgents:
-    def __init__(self, llm):
+    def __init__(self, llm, topic):
         self.llm = llm
+        self.topic = topic
 
     def dlpfc_agent(self):
         return Agent(
             role="Executive Function Manager",
-            backstory=dedent("""
-                Responsible for managing executive functions such as planning and inhibition, essential in complex cognitive tasks.
+            backstory=dedent(f"""
```

Comment:
There is no interpolation in this string so no need for an f-string

```suggestion
            backstory=dedent("""
```